guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,411 @@ Illustration sources:
+   layout = "shadow"
+   caption = "App begins to restore, pulling the channels backup automatically from the cloud."
+   alt-text = "Screen showing an animation as the wallet is being restored"
+   width = 250
+   height = 541
+%}
 
-{% include tip/recommendation.html %}
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   layout = "shadow"
+   caption = "Wallet is finished being restored."
+   alt-text = "Screen showing that the wallet has been restored."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish@2x.png"
+   layout = "shadow"
+   caption = "The wallet has been successfully restored."
+   alt-text = "Screen showing a home screen with funds"
+   width = 250
+   height = 541
+%}
+
+</div>
 
-Be clear to users what you support and what wallet specifications they can expect to restore in your application. Most bitcoin products should support restoring from a recovery phrase at a minimum. Capability to restore an HD wallet implemented according to BIP32, 39, 43, 44, 49, 84 and 380 would be a good starting point.
+{% include tip/open.html color="green" icon="check" label="Use autocomplete" %}
+
+This wallet makes the recovery process easier for the user by autocompleting the words of their recovery phrase as they type. Since recovery phrases are comprised of words from a predefined dictionary of 2,048 words, the app is able usually able to autocomplete or detect a typo with only 3 or 4 letters typed.
 
 {% include tip/close.html %}
 
-Some less popular methods for restoring a wallet include importing a file with all the information required to recover funds such as public keys and key metadata. A wallet.dat file from Bitcoin Core is one example of such a file scheme. If you are developing a watch-only wallet, you would import an [extended public key](/guide/glossary/#extended-public-key-xpub-ypub-zpub) instead of the recovery phrase, this tends to be for technically minded users however.
+## Other situations
+
+### Cloud backup screws up
+
+While this wallet will attempt to pull channel state from a cloud backup, there may be situations in which this does not work automatically. A simple example would be if the user setup the wallet on iOS but is migrating to Android. This could be remedied by allowing the user to specify another cloud account to use.
```

Comment:
```suggestion
While this wallet will attempt to pull channel state from a cloud backup, there may be situations in which this does not work automatically. A simple example would be if the user set up the wallet on iOS, but is migrating to Android. This could be remedied by allowing the user to specify another cloud account to use.
```